db/schema.rb

```diff
@@ -49,7 +49,7 @@
     t.integer "total_applications"
     t.integer "total_awards"
     t.string "collaboration_type"
-    t.integer "oda_eligibility", default: 1, null: false
+    t.integer "oda_eligibility", default: 1
```

Comment:
Are there any places we want to move this not-null validation to? e.g. if it's an ODA/non-ISPF activity, do we want to make `oda_eligibility` required somewhere like the model?

Reply:
Apparently this is already in place! 

```ruby
validates :oda_eligibility, presence: true, on: :oda_eligibility_step, if: :requires_oda_eligibility?

def requires_oda_eligibility?
  !is_non_oda?
end
```

I think the `on: :oda_eligibility_step` means it won't get validated until the form_state is complete, but I think that's fine, because that happens in the importer too.

Reply:
Great, thanks for checking that